code/game/machinery/spaceheater.dm

```diff
@@ -2,7 +2,8 @@
 #define HEATER_MODE_HEAT "heat"
 #define HEATER_MODE_COOL "cool"
 #define HEATER_MODE_AUTO "auto"
-#define BASE_HEATING_ENERGY (STANDARD_CELL_RATE * 0.1)
```

Comment:
Don't change this. STANDARD_CELL_RATE is a global value that is shared by a lot of cell devices.

 Using your own constant value will break the space heater If someone changes it's value in the future.

I didn't mean create your own define, but use this pre existing one